app/views/layouts/_drawer.mobile.haml

```diff
@@ -35,6 +35,13 @@
           = person_image_tag(current_user, size: :thumb_small)
       %li= link_to t("_contacts"), contacts_path
       %li= link_to t("layouts.header.settings"), edit_user_path
+      - if current_user.moderator? && !current_user.admin?
```

Comment:
Sure! Done. I wasn't satisfied by this solution but didn't think about that. I was surprised to discover that the `current_user.moderator?` returns `true` for admins.